source/DateTimePickerBase.js

```diff
@@ -198,7 +198,6 @@
 		*/
 		setupPickers: function (ordering) {
 			// implement in subkind, calling this.inherited() at the end
```

Comment:
I think we'll want to remove this comment if we are no longer calling `this.inherited` from the `setupPickers` methods of subkinds.